pkg/osinserver/registrystorage/storage.go

```diff
@@ -129,21 +130,29 @@ func (s *storage) SaveAuthorize(data *osin.AuthorizeData) error {
 // Client information MUST be loaded together.
 // Optionally can return error if expired.
 func (s *storage) LoadAuthorize(code string) (*osin.AuthorizeData, error) {
-	authorize, err := s.authorizetoken.Get(context.TODO(), code, metav1.GetOptions{})
+	name, prefixed := crypto.TrimSHA256Prefix(code)
+	if prefixed {
+		name = crypto.SHA256Token(name)
+	}
```

Comment:
this part gets repeated often, could we instead have just something like `NormalizePrefixedName` that will return the name to use in client calls?

Reply:
done